x-pack/plugins/apm/server/index.ts

```diff
@@ -116,23 +109,6 @@ export const config: PluginConfigDescriptor<APMConfig> = {
     rename('autocreateApmIndexPattern', 'autoCreateApmDataView', {
       level: 'warning',
     }),
-    renameFromRoot(
-      'apm_oss.transactionIndices',
-      'xpack.apm.indices.transaction',
-      { level: 'warning' }
-    ),
-    renameFromRoot('apm_oss.spanIndices', 'xpack.apm.indices.span', {
-      level: 'warning',
-    }),
-    renameFromRoot('apm_oss.errorIndices', 'xpack.apm.indices.error', {
-      level: 'warning',
-    }),
-    renameFromRoot('apm_oss.metricsIndices', 'xpack.apm.indices.metric', {
-      level: 'warning',
-    }),
-    renameFromRoot('apm_oss.sourcemapIndices', 'xpack.apm.indices.sourcemap', {
```

Comment:
Looks like this one wasn't added to the new plugin config.

Reply:
Good catch! Will add it.

Reply:
Actually, they are right here: https://github.com/elastic/kibana/pull/162367/files#diff-8978eda7925fa795e4e5d209e3cf16fa85885991de0d20f777be26d4b59f3b47R40-R52